civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6553,3 +6553,53 @@ function _civihr_employee_portal_create_default_reports_configuration() {
       ->execute();
   }
 }
+
+/**
+ * Internal function for fetching custom markup from template and Angular Js.
+ *
+ * @param String
+ *    Name of the resource to load template and angular js.
+ * @param String
+ *    Name of the module from where scripts are to be loaded
+ * @param String
+ *    Name of the module from where html is to be loaded
+ *
+ * @return String
+ *    Custom markup from template file.
+ */
+function civihr_get_markup_for_extension($resource = '', $scriptModuleName, $markupModuleName) {
+  $script = CRM_Core_Config::singleton()->debug ? "{$resource}.js" : "{$resource}.min.js";
+
+  // Adds RequireJS + AngularJS dependencies only once
+  if (!strpos(drupal_get_js('footer'), 'reqangular.min.js')) {
+    civicrm_resources_load('org.civicrm.reqangular', ['reqangular.min.js']);
+  }
+
+  civicrm_resources_load($scriptModuleName, [$script]);
```

Comment:
Are you sure this is a module name and not an extension key? On the previous version this used to be an extension key

Reply:
Yes you are right, this should be extension. Thanks